src/CoreBundle/Resources/config/checks.xml

```diff
@@ -46,26 +46,26 @@
         <service id="chameleon_system_core.check_cms_dir_writable"  parent="chameleon_system_sanity_check.check.file_permission">
             <argument>%chameleon_system_sanity_check.level.error%</argument>
             <argument type="collection">
-                <argument>cmsdata</argument>
-                <argument>cache</argument>
-                <argument>cache/autoclasses</argument>
-                <argument>logs</argument>
-                <argument>../web/chameleon/mediapool</argument>
-                <argument>../web/chameleon/outbox/static/css</argument>
-                <argument>../web/chameleon/outbox/static/js</argument>
-                <argument>../web/chameleon/outbox/static/less</argument>
+                <argument>var/cmsdata</argument>
+                <argument>var/cache</argument>
+                <argument>var/cache/autoclasses</argument>
+                <argument>var/log</argument>
+                <argument>public/chameleon/mediapool</argument>
+                <argument>public/chameleon/outbox/static/css</argument>
+                <argument>public/chameleon/outbox/static/js</argument>
+                <argument>public/chameleon/outbox/static/less</argument>
```

Comment:
This can/will be generated automatically if needed - and at a configurable location.